### PR TITLE
[Update] Unread messages intent

### PR DESCRIPTION
**Pull request by AdarshNaidu**
The updated read unread messages intent in this PR can read messages from both public and private channels.

This intent uses simple resolution of channel names. i.e. it will resolve the input to the best matching channel or group name without any intent confirmation.

### CODE REVIEW

**Comment by Wuvist** [comment]
Should updating model with previous intents be in separate PR?

**Review by TheDreamSaver — CHANGES_REQUESTED** [review]
Sanitize the display text that appears on the screen, currently it shows the break tags from SSML as well.

**Review by TheDreamSaver — APPROVED** [review]

**Comment by Sing-Li** [comment]
pls resolve conflict thx

File: models/en-US.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "interactionModel": {
         "languageModel": {
-            "invocationName": "rocket chat",
+            "invocationName": "test rocket",
```
Comment (TheDreamSaver) [inline models/en-US.json]: Change this.

**Review by TheDreamSaver — CHANGES_REQUESTED** [review]

**Review by TheDreamSaver — APPROVED** [review]

**Pull request merged**